File: CurseCS/Model/Angajat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace curse.Model
{
    [Serializable]
    public class Angajat : Entity<long>
    {
        public string Username { get; private set; }
        public string Password { get; private set; }
        public Angajat(string username, string password)
        {
            Id = 0;
            Username = username;
            Password = password;
        }
        public Angajat(long id, string username, string password)
        {
            Id = id;
            Username = username;
            Password = password;
        }
    }
}

File: Curse/src/main/java/Model/Angajat.java
package Model;

public class Angajat extends Entity<Long> {
    public String username;

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public Angajat(String username, String password) {
        this.username = username;
        this.password = password;
    }

    public String password;


    public Object getParola() {
        return this.password;
    }
}
